spec/gutter-spec.coffee

```diff
@@ -0,0 +1,74 @@
+{Gutter, DEFAULT_PRIORITY} = require '../src/gutter'
+
+describe 'Gutter initialization tests.', ->
```

Comment:
We strive to make our specs read in correct english when the words `describe` and `it` are include. For example:

```coffee
describe "Gutter", ->
  describe "::hide", ->
    it "hides the gutter if it is visible", ->
  describe "::show", ->
    it "shows the gutter if it is hidden", ->
  describe "::destroy", ->
    it "removes the gutter from its container", ->
```

So it should very much be structured like "I'm describing this noun. It supports this verb. When I use the verb this is what happens".